web/packages/core/src/ruffle-player.ts

```diff
@@ -721,9 +715,7 @@ export class RufflePlayer extends HTMLElement {
                 const swfDownloadA = document.createElement("a");
                 swfDownloadA.style.display = "none";
                 swfDownloadA.href = blobUrl;
-                swfDownloadA.download = this.swfUrl.substring(
-                    this.swfUrl.lastIndexOf("/") + 1
-                );
+                swfDownloadA.download = swfFileName(this.swfUrl);
```

Comment:
Let's inline `swfFileName` here, like it was before. It's just that I don't see a great value in extracting it to a separate file - no one else needs it at the moment.

Reply:
It was for ease of testing. I couldn't get the test to compile when it imported `ruffle-player.ts`.

Reply:
Hmm, I have mixed feelings about the tests. On one hand it's always a good practice, but on the other hand it's a bit of overkill for this particular relatively simple logic...

Reply:
My threshold for tests is "can you tell it's correct by inspection"? I think in this case, as in most string manipulation cases, I can't be sure without testing it. Also I wouldn't be sure what "pathname" did without looking it up.

Besides, doesn't webpack combine everything into one file on build anyway? So whether it's here or there makes no odds except neatness.